src/sentry/workflow_engine/processors/workflow.py

```diff
@@ -465,6 +466,10 @@ def process_workflows(
     create_workflow_fire_histories(
         detector, actions, event_data, should_trigger_actions, is_delayed=False
     )
+
+    # We only want to fire active actions
+    actions = actions.filter(status=ObjectStatus.ACTIVE)
```

Comment:
Should this be done in `fire_actions` or `deduplicate_actions` inside of it? `fire_actions` is used in both process_workflows and delayed workflow processing

Reply:
I moved `deduplicate_actions` inside `fire_actions` before. I don't think it hurts because you should only fire active actions anyway